ministryofjustice/correspondence_tool_staff
# CT-3925: Add sar ir subtype to report

## Description
Amends the cases reports so as allow for SAR IRs. Adds new column to reports to show whether a SAR IR is for 'timeliness' or 'compliance'.

New column title is: 'SAR IR - timeliness or compliance'

## Self-review checklist


### Related JIRA tickets
[CT-3925](https://dsdmoj.atlassian.net/browse/CT-3925)

### Manual testing instructions
- you should already be able to search of SAR IR cases, so please check that you can.
- download the open cases report - you should be able to see a new column in the report that reports the SAR IR subtypes of 'timeliness' or 'compliance'
